scilpy/segment/tractogram_from_roi.py

```diff
@@ -123,7 +126,7 @@ def _extract_and_save_tails_heads_from_endpoints(gt_endpoints, out_dir):
     affine = mask_img.affine
     dimensions = mask.shape
 
-    head, tail = split_heads_tails_kmeans(mask)
+    head, tail = split_mask_blobs_kmeans(mask, nb_clusters=2)
```

Comment:
Why the extra parameter ? Does it change the behaviour of the function ?

Reply:
No. I just moved the function to image.utils because it only takes a mask as parameter, no tractogram. So it felt weird to keep in named split_head_tail. It could split any mask into two blobs. And why not into K blobs, if eventually anyone needs it. So it went from:

Tractanalysis.tools:
```
    X = np.argwhere(data)
    k_means = KMeans(n_clusters=2).fit(X)
    mask_1 = np.zeros(data.shape)
    mask_2 = np.zeros(data.shape)

    mask_1[tuple(X[np.where(k_means.labels_ == 0)].T)] = 1
    mask_2[tuple(X[np.where(k_means.labels_ == 1)].T)] = 1
```
To

image.utils:
```
    X = np.argwhere(data)
    k_means = KMeans(n_clusters=nb_clusters).fit(X)

    masks = []
    for i in range(nb_clusters):
        mask_i = np.zeros(data.shape)
        mask_i[tuple(X[np.where(k_means.labels_ == i)].T)] = 1
        masks.append(mask_i)

    return masks
```

But I forgot to change the doctstring. Doing it now, thanks.